src/proc_loadavg.c

```diff
@@ -138,7 +138,10 @@ static void insert_node(struct load_node **n, int locate)
 {
 	struct load_node *f;
 
-	pthread_mutex_lock(&load_hash[locate].lock);
+
+	if(mutex_lock_interruptible(&load_hash[locate].lock)){
```

Comment:
what's the purpose replacing of a non-interruptible mutex with interruptible in this case?

Also, if you doing that, you need to handle "interrupt" case very-very carefully in all places from where the surrounding function (`insert_node`) is called.